app/grandchallenge/pages/views.py

```diff
@@ -86,6 +88,48 @@ def test_func(self):
         return page.can_be_viewed_by(user=user)
 
 
+class PagePandoc(
+    UserPassesTestMixin, ChallengeFilteredQuerysetMixin, DetailView
+):
+    model = Page
+    permission_required = "view_page"
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
+
+    raw = None
+
+    def test_func(self):
+        return self.request.user.is_staff
+
+    def get_object(self, *args, **kwargs):
+        if not self.kwargs["format"].startswith("markdown"):
+            raise PermissionDenied()
```

Comment:
Why have a `format` variable if we only allow one format at the moment?

Reply:
Because there are many flavours of MarkDown that I would like to try: https://pandoc.org/chunkedhtml-demo/8.22-markdown-variants.html

Reply:
Ah I misread this check. This is already allowing multiple formats, so long as they start with `markdown`.